packages/e2e-tests/specs/editor/various/keyboard-navigable-blocks.test.js

```diff
@@ -103,21 +85,20 @@ describe( 'Order of block keyboard navigation', () => {
 		for ( const paragraphBlock of paragraphBlocks ) {
 			await insertBlock( 'Paragraph' );
 			await page.keyboard.type( paragraphBlock );
-			await page.keyboard.press( 'Enter' );
 		}
 
-		await navigateToContentEditorTop();
+		// Select the middle block.
+		await page.keyboard.press( 'ArrowUp' );
+		// Move the mouse to show the block toolbar
+		await page.mouse.move( 0, 0 );
+		await page.mouse.move( 10, 10 );
```

Comment:
It troubles me when I see interactions like these in the end-to-end tests, because the first question which comes to mind is: Is there not an easier way to express this as a keyboard interaction? Or, worse, is it _even possible_ to achieve this behavior via keyboard alone?

I'm not sure if it matters much for this specific test, since it seems to be at least somewhat contrived example to verify which options are present, but in general I'm a bit concerned about navigation mode, or at least specifically options for "cancelling" the `isTyping` flag. Previously, it was problematic in how it relied on a mouse or clunky selection operations to control. Later, <kbd>Escape</kbd> was added as an option to unset the flag (#10906), but I guess now that's been overridden to control whether the block editor is in navigation mode. I worry this all has put us back in a position we were in prior to #10906 where certain interactions are only possible using the cursor.